plugin.js

```diff
@@ -464,8 +465,9 @@ function makeWorklet(t, fun, fileName) {
     t.expressionStatement(
       t.assignmentExpression(
         '=',
-        t.memberExpression(privateFunctionId, t.identifier('asString'), false),
-        t.stringLiteral(funString)
+        t.memberExpression(privateFunctionId, t.identifier('__reanimated_workletFunction'), false),
+        // t.stringLiteral(funString)
+        t.callExpression(t.identifier('eval'), [t.stringLiteral('(' + funString + ')')])
```

Comment:
The call to `eval` here is always done with the result of compiling the AST. This shouldn't present a security risk, but definitely looks ugly. The `eval` call was added as a workaround to prevent the function code to be rewritten by other babel plugins. There could be better ways to solve this.